rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java

```diff
@@ -99,19 +110,41 @@ public static void main(String[] args) throws Exception {
     public static class EnrichmentFunction
             extends RichCoFlatMapFunction<TaxiRide, TaxiFare, RideAndFare> {
 
+        private ValueState<RideAndFare> rideAndFareValueState;
```

Comment:
it has to be `transient`, please modify it to avoid Serialization/Deserialization issues

Reply:
done